runtime/default.policy

```diff
@@ -94,13 +94,17 @@ rules:
       open.filename =~ "/etc/ssl/certs/*" && open.flags & (O_CREAT | O_RDWR | O_WRONLY) > 0
     tags:
       technique: T1338
-  - id: pci_11_5
+  - id: pci_11_5_bin
```

Comment:
I think, that it could be renamed as pci_11_5_critical_binaries. The rule description can also be updated to reflect that the rule is specific to binaries.

Reply:
👍